ee/packages/pdf-worker/src/strategies/ChatTranscript.ts

```diff
@@ -1,5 +1,7 @@
+import { Translation as translationService } from '@rocket.chat/core-services';
```

Comment:
since this is a "pdf" package, i wonder if we should do the system message parsing on the omnichannel-transcript service instead of inside pdf-worker package.